mappings/net/minecraft/entity/ai/brain/task/TakeJobSiteTask.mapping

```diff
@@ -0,0 +1,20 @@
+CLASS net/minecraft/class_5327 net/minecraft/entity/ai/brain/task/TakeJobSiteTask
+	FIELD field_25158 speed F
+	METHOD <init> (F)V
+		ARG 1 speed
+	METHOD method_29260 canUseJobSite (Lnet/minecraft/class_4158;Lnet/minecraft/class_1646;Lnet/minecraft/class_2338;)Z
+		ARG 1 poiType
```

Comment:
I think we used `poi` in other classes

Reply:
We do for `PointOfInterestType`? I think we use poi for poi instances, do we?

Reply:
Yeah `PointOfInterestType` is like `StatusEffect` as `PointOfInterest` is to `StatusEffectInstance`